banner_features.yaml

```diff
@@ -0,0 +1,165 @@
+desktop_banner: # npm run dandy -- -b B22_WMDE_Test_27_ctrl -d -h desktop_banner
```

Comment:
I suggest adding an optional 'extends' section to the features here that would allow a feature set to include another's without having to copy and paste all the test names. Then a class could be made (maybe FeatureCollector) that will recursively loop the feature sets and collect all the extended tests.

A check to see if the same feature set is extended twice might be needed to prevent accidental infinite recursive loops.

Here's a quick example:

```js
export default class FeatureCollector {
    featureSets;
    visitedFeatureSets = [];

    construct(featureSets) {
        this.featureSets = featureSets;
    }

    collectFeatures(featureSetName: string): string[] {
        if(visitedFeatureSets.includes(featureSetName)) {
            // throw an exception to stop features infinite looping
        }
        visitedFeatureSets.push(featureSetName);
        if(featureSets[featureSetName].extends) {
            return this.collectFeatures(featureSets[featureSetName].extends);
        }
        return featureSets[featureSetName].features;
    }
}

const features  = (new FeatureCollector(featureSets)).collectFeatures('mobile_banner_var');

Reply:
Oops, forgot to add something that merges the returned array from the extended